src/identity/rolesync.service.ts

```diff
@@ -39,6 +45,82 @@ export class RoleSyncService {
     private readonly membershipsProvider: CacheableMembershipsProvider,
   ) {}
 
+  @Once('ready')
+  async syncActiveServerswithDB() {
+    const servers = await this.client.guilds.fetch();
+
+    this.logger.log('Start syncing active servers with DB...');
+
+    const promises = servers.map((server) => this.syncServerRoles(server.id));
+
+    await Promise.all(promises);
+
+    this.logger.log('Done syncing active servers with DB ✅');
+  }
+
+  private async syncServerRoles(serverId: string) {
+    const server = await this.client.guilds.fetch(serverId);
+
+    this.logger.log(`Start syncing '${server.name}' with DB...`);
+
+    const serverUsers = await server.members.fetch();
+
+    const serverIdentityValidatedRole = await findServerRole(
+      this.client,
+      serverId,
+      identityValidatedRole,
+    );
+
+    if (!serverIdentityValidatedRole) {
+      this.logger.error(
+        `The '${identityValidatedRole}' role wan't found in the ${server.name} (Server ID: ${server.id})`,
+      );
+      return;
+    }
```

Comment:
Example how to make code more readable.
```suggestion
    if (!serverIdentityValidatedRole) {
      const message = `The '${identityValidatedRole}' role wasn't found in the ${server.name} (Server ID: ${server.id})`
      return this.logger.error(message)
    }
```